deploy/connect-msk.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: playbook-dispatcher-connect
 parameters:
 - name: KAFKA_CONNECT_IMAGE
-  value: quay.io/cloudservices/playbook-dispatcher-connect
+  value: quay.io/redhat-services-prod/hcc-integrations-tenant/playbook-dispatcher-connect
```

Comment:
This has no immediate effect because the image path is already set to the same value through the `KAFKA_CONNECT_IMAGE` param in app-interface.